src/api/channels.js

```diff
@@ -1,13 +1,14 @@
 import logger from 'winston'
 import request from 'request'
+
 import * as Channels from '../model/channels'
```

Comment:
That might break tests as deconstructed imports can't be mocked :(

https://www.exratione.com/2015/12/es6-use-of-import-property-from-module-is-not-a-great-plan/

But perhaps we can have a boyscouting ticket that deals with making the import mechanism consistent - this will probably have to remove all the restructuring to allow proper testing (even though I like the neatness of it)

Reply:
ok cool i think thats fine. Lets do this as part of a different task in future then :+1